underscore.js

```diff
@@ -1279,9 +1288,7 @@
 
   // Is a given value an array?
   // Delegates to ECMA5's native Array.isArray
-  _.isArray = nativeIsArray || function(obj) {
-    return toString.call(obj) === '[object Array]';
-  };
+  _.isArray = isArray;
```

Comment:
Can we remove or at least change the eslint rule? It's totally valid to lazily define methods on the `_` object provided they're not getting used right away.